manifests/install.pp

```diff
@@ -8,8 +8,45 @@
   # this is a private class
   assert_private("You're not supposed to do that!")
 
-  package{'ferm':
-    ensure => 'latest',
+  case $ferm::install_method {
+    'package': {
+      package{'ferm':
+        ensure => 'latest',
+      }
+    }
+    'vcsrepo': {
+      $_source_path = '/opt/ferm'
+
+      package{'ferm':
+        ensure => absent,
+      }
+
+      -> package { ['iptables', 'perl', 'make']:
```

Comment:
at least perl and make could be managed by another module, should be use ensure_package here?

Reply:
list of managed packages is now an `Array` as parameter

Reply:
The function [ensure_packages()](https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/REFERENCE.md#ensure_packages) is now used.